.travis.yml

```diff
@@ -1,15 +1,14 @@
-os:
-- osx
+os: osx
+osx_image: xcode9.2
 script:
-- rm -rf /usr/local/Homebrew
-- travis_wait 60 rvm implode --force
-- ./script/boxen --no-fde --stealth --no-pull --debug --profile --login $BOXEN_GITHUB_LOGIN --token $BOXEN_GITHUB_TOKEN || test $? -eq 2
-- source /opt/boxen/env.sh
-- bundle exec rspec spec
+  - sudo rm -rf /usr/local/Homebrew
+  - travis_wait 60 sudo rvm implode --force
+  - ./script/boxen --no-fde --no-pull --debug --profile --login $BOXEN_GITHUB_LOGIN --token $BOXEN_GITHUB_PASSWORD || test $? -eq 2
+  - source /opt/boxen/env.sh
+  - bundle exec rspec spec
 before_install:
-- sudo gem install bundler -v '~> 1.13'
+  - sudo gem install bundler -v '~> 1.10.5'
```

Comment:
Any reason why we're going back to an older version of bundler?

Reply:
Now bundler bootstrap script version is `1.10.5`

https://github.com/boxen/our-boxen/blob/master/script/bootstrap#L57